src/index.ts

```diff
@@ -40,7 +40,5 @@ export * from "./IClientOptions";
 export * from "./IContext";
 export * from "./IFetchOptions";
 export * from "./IGraphRequestCallback";
-export * from "./ImplicitMSALAuthenticationProvider";
```

Comment:
This change would be breaking to some people, depending on their import style. Are we planning to major bump?

Reply:
It is a major bump.